components/properties/src/props.rs

```diff
@@ -891,6 +897,427 @@ impl Script {
 }
 }
 
+impl Script {
+    // Doesn't actually exist!
+    #[doc(hidden)]
+    #[allow(non_upper_case_globals)]
+    #[deprecated]
+    pub const Chisoi: Script = Self(254);
+}
+
+#[cfg(feature = "compiled_data")]
+impl From<Script> for icu_locale_core::subtags::Script {
+    fn from(value: Script) -> Self {
+        crate::PropertyNamesShort::new()
+            .get_locale_script(value)
+            .unwrap_or(icu_locale_core::subtags::script!("Zzzz"))
+    }
+}
+
+#[cfg(feature = "compiled_data")]
+impl From<icu_locale_core::subtags::Script> for Script {
+    fn from(value: icu_locale_core::subtags::Script) -> Self {
+        crate::PropertyParser::new()
+            .get_strict(value.as_str())
+            .unwrap_or(Self::Unknown)
+    }
+}
```

Comment:
I've wanted this conversion for a very long time. I think I tried adding it before and the PR got blocked for some reason.

Reply:
for one we cannot add a method on the locale `Script` from this crate, so we cannot have symmetric bidirectional conversions. `From` is the idiomatic Rust way for type conversions, and it allows this.